PubnativeLite/PubnativeLite/Interstitial/HyBidInterstitialAd.h

```diff
@@ -60,4 +61,6 @@
 - (void)showFromViewController:(UIViewController *)viewController;
 - (void)hide;
 
+- (void)setSkipOffset:(NSInteger)seconds;
```

Comment:
Do you have to create an extra method for setting the `skipOffset`? I think you can directly use the property since it is not `readonly`. i.e `self.interstitialAd.skipOffset = 3.0`

Reply:
Here I agree that we should either use the property but with a proper setter, because it needs validations, or use the method and make the property private.